importers/src/bible/osis/index.ts

```diff
@@ -468,14 +440,14 @@ export class OsisImporter extends BibleEngineImporter {
                         currentContainer.contents.push(paragraph);
                     } else currentContainer.contents.push(titleGroup);
                     context.contentContainerStack.push(titleGroup);
+                } else if (!this.hasSectionsInText) {
+                    // Since titles should always be attached to a section,
+                    // versions with titles but not sections need artifical sections
+                    this.startNewSection(context, OsisXmlNodeType.SECTION)
```

Comment:
didn't look at this in detail, but do you guard this against canonical titles (that don't start sections since they are part of the text and sections are not)

Reply:
good question, yes I have a guard for that